cypress/e2e/carsTest.cy.js

```diff
@@ -0,0 +1,64 @@
+import * as testData from '../fixtures/testData.json'
+
+describe('Cars functionality test', () => {
+
+  beforeEach(() => {
+    cy.visit('/', {headers: {"Accept-Encoding": "gzip, deflate"}})
+  })
+
+  it('comparison for two cars can be performed', () => {
+    cy.checkMainPageIsOpen()
```

Comment:
It is not necessary to make space after each step, we will end up with a long test files.